### ISSUE TITLE
Find Next Similar Item

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#20055]
### Your idea

Maybe this could run from the right click menu where there is currently an option to select all similar items and the associated dialogue box under more that allows you to specify what and where it is to be looked for.

But in this case it allows you to quickly move with a keyboard shortcut to each similar item.

### Problem to be solved

This would allow a blind user a quick way to for example review all the dynamic markings in a piece, or jump to each ornament.  currently these can be very hard to find when you have to arrow over each note and wait to hear the note before you know if it has any other element attached.  I'm sure it could be a helpful tool for sighted users also when editing all of a similar item in a score.

### Prior art

Yes, Lime (GoodFeel Suite) has an option to navigate along just the annotations, although this would be even better if it could be separated into specific elements rather than all annotations.

**Comment by oktophonie** [comment]
Related to this, SCORE (of blessed memory) had an command that would let you step through all items of a given type on a system, and I used this all the time as it was very useful (e.g. to step through all the slurs, or notes, or whatever).

**Comment by avvvvve** [comment]
For anyone who would like to work on this, at minimum we could add these two new actions to Preferences > Shortcuts:

- Go to next similar element
- Go to previous similar element

They don't need to have shortcuts by default. For simplicity, if you're navigating with this shortcut from one text element to another, this shortcut should _not_ enter edit mode and the shortcut probably should not work while you're editing text (because, for example, if the user set it to something like `Shift` + `J` we'd have to make a decision whether to type a capital J or use the shortcut).

**Comment by jmwmusic** [comment]
@avvvvve Agree, text fields and any other editable elements should be focused to jump to with the shortcut, but not automatically editable as that would also break navigation.  We can use F2 or Enter to open edit as usual.  users may well use this shortcut in combo with other navigation commands to move around quickly to a new section, phrase etc that they know has a dynamic mark for example, so don't want to then find themselves stuck in an open editable field when they go to arrow.
